timed/employment/models.py

```diff
@@ -259,54 +260,11 @@ def calculate_worktime(self, start, end):
         :returns:     tuple of 3 values reported, expected and delta in given
                       time frame
         """
```

Comment:
The docstring does not seem to fit the function anymore since you moved the code to another function that is called by it.

I wonder if it would be sufficient to create the part for the employment.

But I'm not familiar enough with the model situation here e. g. why the calculation is bound to the Location model in the first place..